src/cmd/Makefile.am

```diff
@@ -85,7 +85,9 @@ dist_fluxcmd_SCRIPTS = \
 	flux-job-exec-override.py \
 	flux-perilog-run.py \
 	flux-uri.py \
-	flux-pstree.py
+	flux-pstree.py \
+	flux-pgrep.py \
+	flux-pkill.py
```

Comment:
When installed, the symlink is lost and two copies of the executable are installed.   In other projects I've seen `install-exec-hook` used to create symbolic links, as described in the autotools docs:

https://www.gnu.org/software/automake/manual/automake.html#Extending

I'm not sure if it's worth the extra complexity honestly, but thought I'd mention it.

Reply:
Thanks I looked into that when doing the original `pgrep` PR but I can't remember why I discarded it. I'll take another look. FWIW, the main `pgrep`/`pkill` package also installs a copy and not a symlink...

Reply:
Took another look and I think I've got this working. It is unfortunately quite a bit more complex since you need both an install and uninstall-local hook, plus a rule to generate the flux-pkill.py symlink locally so that `flux pkill` works in-tree. However, we don't then have to commit a symlink to git so I guess that's better?

Reply:
Sorry about that, I just wasn't sure what the current best practice was for that sort of thing.  Whichever way makes you happy is OK with me.